lib/play/game_page.dart

```diff
@@ -542,6 +542,11 @@ class _GamePageState extends State<GamePage> {
 
     // Send automatic counting command to game client
     widget.game.automaticCounting().then((info) {
+      if (info.timeout == Duration.zero) {
```

Comment:
I'm sure this means I am abusing the automaticCounting APIs 😅 

But essentially, I found that progress bar modal didn't make sense in OGS context, and it also prevented the user from taking other actions.